woocommerce/woocommerce
# Product Hero Pattern: Fix button alignment

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Wraps button in a group block for WooCommerces Product Hero pattern which fixes the button alignment issue and renders it left aligned rather than center.



Closes https://github.com/woocommerce/woocommerce/issues/42189



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Confirm issue on `trunk` before checking out the branch.
2. Checkout branch and add the `Product Hero` pattern into a post or page.
3. Ensure the Add to Cart button is left aligned rather than center aligned.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Fixed the alignment issue of the Add to Cart button in WooCommerces Product Hero pattern.

#### Comment 

</details>
